common/web/types/package.json

```diff
@@ -79,5 +82,6 @@
       "src/schemas/*",
       "test/"
     ]
-  }
+  },
+  "sideEffects": false
```

Comment:
Without this specific line, a _lot_ of the package fails to tree-shake out.  Notably, all of the precompiled schemas get included, which would cause major filesize bloat.